packages/bee-q/src/components/notification/bg-notification-types.ts

```diff
@@ -0,0 +1 @@
+export const NOTIFICATION_TYPES = ['info', 'success', 'warning', 'error'] as const;
```

Comment:
I would recommend adding a `default` type as well, were the icon would not take any specific color.

```suggestion
export const NOTIFICATION_TYPE = ['default', 'info', 'success', 'warning', 'error'] as const;
export type TNotificationType = (typeof NOTIFICATION_TYPE)[number];
```

and then you would be able to set in the component implementation:

```ts
@Prop({ reflect: true }) type: TNotificationType = 'default';
```